### PR TITLE
fix(mql): Have MQL use double instead of single quotes

### PR DESCRIPTION
**Pull request by evanh**
MQL is meant to align with Discover syntax as much as possible. Discover uses
double quotes instead of single quotes, so when a MetricsQuery is serialized it
needs to be serialized using double quotes instead of single quotes.

### CODE REVIEW

**Comment by volokluev** [comment]
Do we need to change the snuba code as well?

**Comment by evanh** [comment]
> Do we need to change the snuba code as well?

@volokluev Yes, and I will open that PR once this is merged. I can't open an official PR until this is released.

File: snuba_sdk/visitors.py [hunk 1]
```diff
@@ -122,8 +122,9 @@ def _visit_totals(self, totals: bool) -> TVisited:
 
 
 class Translation(ExpressionVisitor[str]):
-    def __init__(self, use_entity_aliases: bool = False):
+    def __init__(self, use_entity_aliases: bool = False, is_mql: bool = False):
```
Comment (volokluev) [inline snuba_sdk/visitors.py:125]: What can it be besides `mql`? In which cases would `is_mql` be False?

**Review by volokluev — COMMENTED** [review]
Is it correct to say that the grammar already uses double quotes and we just need to change the mql generation?

File: snuba_sdk/visitors.py (hunk 1 above)
Comment (evanh) [reply snuba_sdk/visitors.py:125]: This is used to encode both SnQL and MQL. In SnQL we use single quotes, `array(...)` and `tuple(...)`. In MQL we use double quotes and `[...]`.

**Review by volokluev — APPROVED** [review]

**Pull request merged**